<launch>

    <arg name="project" default="LiDAR_Inertia_Localiztion"/>


    <node pkg="$(arg project)" type="LiDAR_matching" name="LiDAR_matching" output="screen" args="0.0375">
<!--        <remap from="/velodyne_points" to="/livox/lidar"/>-->
<!--        <remap from="/imu/data_raw" to="/livox/imu"/>-->
    </node>
<!--    <node pkg="$(arg project)" type="IMU_Preintegration" name="IMU_Preintegration"   output="screen"  respawn="true"/>-->
    <node pkg="rviz" type="rviz" name="$(arg project)_rviz" args="-d $(find LiDAR_Inertia_Localiztion)/launch/rviz.rviz" />

</launch>